src/pages/hegel/guides/science-of-logic-introduction.mdx

```diff
@@ -0,0 +1,184 @@
+
+### Dialectic: The Engine of Logic
+
+If the method of logic is something that emerges, develops and evolves, one is instinctively prompted by the question of _what_ drives logic itself to progress in the first place? Keeping in mind everything said about indications of the content above, Hegel's simple reply to that question is:
+
+> What propels the concept onward is the already mentioned negative which it possesses in itself; it is this that constitutes the truly dialectical factor (Hegel, 2010, 34/21.39).
+
+Here there are two elements which, although often intertwined, are subtly distinct: the negative and the dialectic. The dialectic has a rich lineage in the history of philosophy and Hegel draws some key comparisons between his usage and that of Plato and Kant. Before turning to that discussion, it is worthwhile to keep in mind how the negative and the dialectic are distinct. 
+
+While no detailed examination can take place of these terms here, some cursory remarks and images can be helpful. When speaking of Hegel's dialectic, consider the negative as a moment of something that activates its transformation into something other. What is makes this change particularly _dialectical_ is the fact that the change is purely internal. The thing in question simply changes on its own account, like a piston engine that is able generate its own pressure to create motion, or, perhaps more appropriately, a heart that generates the pulse necessary for the support system of the heart to in turn keep beating. Now, in absolutely strict terms, one might determine that what the thing first is and then becomes are essentially two different things, such that there is really no change, much less dialectic, since the thing is no longer the same thing should there be any new anything. So it is required inasmuch as something is understood dialectically that the thing in question minimally retains an identity in spite of difference. Moreover, in thinking of dialectic as change, this change entails many categories working together: constitutive moments, spontaneous or self-determined negative moment, integration of that negative moment into a whole. One can see how this idea quickly reveals a wealth of determinations. 
```

Comment:
"question minimally retains an identity in spite of difference." - I wonder if it would be helpful to include sublation at this point since it is not 100% clear why the identity should be retained given what you just said about there being 2 different things. Sublation would help to explain why we continue to think of them minimally identical.

Reply:
yeah, but i think sublation should be saved for its own article. this is already quite long. In the future we can drop a link here to "sublation" article once that's up.